Makefile

```
@@ -177,12 +172,16 @@ golangci-lint-github-action:
 cover:
 	@echo "mode: count" > cover-all.out
 	@export CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)";\
-	  $(foreach pkg,$(PKGS),\
+	  $(foreach pkg,$(UNIT_TESTS),\
 		go test -coverprofile=cover.out -covermode=count $(pkg);\
 		tail -n +2 cover.out >> cover-all.out;)
 	go tool cover -html=cover-all.out
 .PHONY: cover
 
+cover-profile:
+	@$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" go test -coverprofile=cover.out -timeout 0 -p 1 $(UNIT_TESTS)
+.PHONY: cover-profile
```

It's not clear to me why there is a second coverage target. Could you please leave a comment in the Makefile justifying it?

Btw, it would be super nice to run coverage in CI 😉.

---

I'm not 100% sure what the other target is for. It uses `covermode=count` instead of the default `covermode=set` and it iterates over all packages manually, instead of just making one call.

What it is probably used for can be seen by the last line: it starts a server that visualizes the generated coverprofile in the browser.

The `cover.out` file created by the new `cover-profile` target is just a file that can be loaded in VScode to visualize which codepaths are covered (green background) and which are not (red background)

---

Perhaps let's initiate a discussion to see if anybody actually uses the `cover` target. Then:
- if nobody uses it - remove it,
- if somebody uses it - we will know why

@dshulyak @countvonzero @evt What do You think?

---

Alternatively these targets can also be cleaned up nicely (assuming `covermode=set` works for the HTML tool as well as `covermode=count`.

```suggestion
cover.out:
	@$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" go test -coverprofile=cover.out -timeout 0 -p 1 $(UNIT_TESTS)
	
cover-profile: cover.out
.PHONY: cover-profile
	
cover-tool: cover.out
	@go test cover -html=cover.out
.PHONY: cover-tool

clear-cover-profile:
	@rm cover.out
.PHONY: clear-cover-profile
```